frontend/src/components/SearchResultInfo.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router-dom';
+
+type Props = {
+  total: number;
+  city: string;
+};
+
+const SearchResultInfo = ({ total, city }: Props) => {
+  return (
+    <div className="text-xl font-bold flex flex-col gap-3 justify-between lg:items-center lg:flex-row">
+      <span>
+        {total} {total === 1 ? 'Restaurant' : 'Restaurants'} found in {city}
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/43#issuecomment-2408154714>#86982f</a><span> - 10/11/2024, 09:46 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The component lacks proper accessibility attributes, which may make it difficult for screen readers to interpret the content correctly.
</div>
<div id="fix">
<b>Fix:</b> Add appropriate ARIA attributes to improve the component's accessibility. Use aria-live for dynamic content and aria-label for the link.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -10,13 +10,13 @@
    return (
 -    <div className="text-xl font-bold flex flex-col gap-3 justify-between lg:items-center lg:flex-row">
 -      <span>
 +    <div className="text-xl font-bold flex flex-col gap-3 justify-between lg:items-center lg:flex-row" aria-live="polite">
 +      <span aria-label={`${total} ${total === 1 ? 'Restaurant' : 'Restaurants'} found in ${city}`}>
          {total} {total === 1 ? 'Restaurant' : 'Restaurants'} found in {city}
          <Link
            to="/"
            className="ml-1 text-sm font-semibold underline cursor-pointer text-blue-500"
 +          aria-label="Change search location"
          >
            Change Location
          </Link>
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged